### ISSUE TITLE
View.mouseWheelAction_ doesn't always receive xDelta or yDelta

### ISSUE THREAD

**Issue by jrsurge** [supercollider/supercollider#4422]
## Environment

* SuperCollider version: 3.10.2
* Operating system: Windows 10, 1809, 17763.504
* Other details (Qt version, audio driver, etc.): Qt 5.11.2

## Steps to reproduce
1) Run:
```supercollider
(
Window().asView.mouseWheelAction_({ | view, x, y, modifiers, xDelta, yDelta | 
	[xDelta, yDelta].postln;
}).front;
)
```

2) Scroll inside the Window with the mouse.

## Expected vs. actual behavior

Expected to post scroll delta values.
Actual:
Posts `[0.0, 0.0]` - delta values are always 0.0

**Comment by jrsurge** [comment]
This appears to be another Windows issue.

I think I have a fix, but need to do some testing to make sure it's consistent across platforms

**Comment by brianlheim** [comment]
Closed in #4423, thanks!
